### PR TITLE
Updates accents on céntimos in Spanish

### PR DESCRIPTION
**Pull request by ohduran**
### Changes proposed in this pull request:

* Fixes Spanish language

### Status

- [X] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### How to verify this change

This is a cosmetic change: make sure that the accent in 'céntimos' is there when running num2words with `to='currency'`.

### CODE REVIEW

**Comment by erozqba** [comment]
@ohduran thanks for this contribution! You forgot to update the tests, please update them too.

**Comment by erozqba** [comment]
@ohduran thanks for updating the tests, please take a look into `test_cli_with_lang_to` because looks like Spanish is also used on this test and need to be updated.

**Comment by ohduran** [comment]
@erozqba I'm unable to bring this forward; tests pass when using the command, but tox keeps on failing. Could you please point me in the right direction?

Cheers,
Alvaro.

File: tests/test_cli.py [hunk 1]
```diff
@@ -106,6 +106,6 @@ def test_cli_with_lang_to(self):
         output = self.cli.run_cmd(150.55, '--lang', 'es', '--to', 'currency')
         self.assertEqual(output.return_code, 0)
         self.assertEqual(
-            output.out.strip(),
-            "ciento cincuenta euros con cincuenta y cinco centimos"
+            output.out.strip().decode('utf-8'),
```
Comment (kingbuzzman) [inline tests/test_cli.py]: Would it make sense to add `locale.getpreferredencoding()` here instead of `utf-8`?

**Comment by kingbuzzman** [comment]
@ohduran @erozqba 

When running it locally i can get it to reproduce by doing:

`PYTHONIOENCODING=ascii python /num2words/bin/num2words 150.55 --lang es --to currency`

It seems like `delegator.Command` is setting unsetting the `PYTHONIOENCODING` and causing it to crash. The issue with what do do when its ascii remains open.

**Comment by ohduran** [comment]
Hi @erozqba, waiting for your approval to merge this PR and have the grammar corrected. Do let me know if there is anything else you need me to do.

Alvaro.

**Comment by kingbuzzman** [comment]
@erozqba can you make a release for 0.5.10 with this in it? Or are you waiting on some other milestone?

**Pull request merged**